web/docs/language/overview.md

```diff
@@ -7,11 +7,11 @@ words (types) of the language.
 
 The basic idea is that the higher-level overview of an app (e.g. pages, routes, database model, ...) is defined in `*.wasp` files (for now just one), while the specific parts (web components, back-end queries, ...) are implemented in specific non-Wasp technologies (React, NodeJS, Prisma) and then referenced in the `*.wasp` files.
 
-Basic structure of a Wasp project is:
+The basic structure of a Wasp project is:
 - `*.wasp` file
-- The `src/` folder -> Contains non-Wasp code (JS, CSS, ...). You can structure it however you want, as long as you put it somewhere inside the correct subfolder:
+- The `src/` folder -> Contains non-Wasp code (JS, CSS, ...). You can structure it, however, you want, as long as you put it somewhere inside the correct subfolder:
   - The `src/server` folder - Contains your server code (i.e., executed by Node JS).
-  - The `src/client` folder - Contains your client code (i.e., executed by JS in user's browsers).
+  - The `src/client` folder - Contains your client code (i.e., executed by JS in the user's browsers).
```

Comment:
Grammarly misunderstood what was going on here, it's supposed to be either "in users'  browsers" or "in the user's browser"

Reply:
Makes sense, I should have doubled checked Grammarly a bit more.